src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -107,7 +107,16 @@ protected override void EndProcessing()
                 object objectToProcess = (_inputObjects.Count > 1) ? (_inputObjects.ToArray() as object) : (_inputObjects[0]);
                 // Pre-process the object so that it serializes the same, except that properties whose
                 // values cannot be evaluated are treated as having the value null.
-                object preprocessedObject = ProcessValue(objectToProcess, 0);
+                object preprocessedObject = null;
+                try
+                {
+                    preprocessedObject = ProcessValue(objectToProcess, 0);
+                }
+                catch (StoppingException)
+                {
+                    return;
+                }
+                CheckStopping();
```

Comment:
I am a little confused by this.  The StoppingException is caught and ignored in one case but allowed to throw if ProcessValue() completes via CheckStopping().  What is the user experience?  Is the user intended to see the exception or does the cmdlet just stop?

Reply:
I don't think we need to check for stopping again here. You may argue that the user can have a final chance to stop the command before the `JsonConvert.SerializeObject` has the control, though :)

If you really want to check for stop another time here, then `if (Stopping) { return }` should be sufficient.

Reply:
Yes, will change.